packages/lib-classifier/src/store/TranscriptionReductions/TranscriptionReductions.js

```diff
@@ -0,0 +1,119 @@
+import asyncStates from '@zooniverse/async-states'
+import { GraphQLClient } from 'graphql-request'
+import { flow, types } from 'mobx-state-tree'
+
+export const caesarClient = new GraphQLClient('https://caesar.zooniverse.org/graphql')
+const CONSENSUS_SCORE_TO_RETIRE = 3
+const MINIMUM_VIEW_TO_RETIRE = 5
```

Comment:
Are these parameters the same for all transcription projects?

Reply:
They were fixed for ASM, but we are looking to make it a bit more general. At the moment `CONSENSUS_SCORE_TO_RETIRE` can be set directly on the reducer as the `low_consensus_threshold` parameter. The reducer uses this threshold to sets the boolean flag `low_consensus` for each line.  

`MINIMUM_VIEW_TO_RETIRE` could be set in a similar way, but isn't at the moment (I will open an issue for this).

Reply:
See https://github.com/zooniverse/aggregation-for-caesar/issues/259

Reply:
Thanks, so we can expect to find these parameters listed in the response to the GraphQL query?

Reply:
I've updated the PR to read the consensus score from `reduction.data.parameters.low_consensus_threshold` and fall back to `CONSENSUS_SCORE_TO_RETIRE`.

Reply:
Once https://github.com/zooniverse/aggregation-for-caesar/pull/261 is merged `reduction.data.parameters.minimum_views` will also be available.

Reply:
Great! I’ll add that in too, with a fallback to the constant.